.github/workflows/publish-jvm.yaml

```diff
@@ -43,7 +46,7 @@ jobs:
   build-jvm-full-library:
     name: Create full bdk-jvm library
     needs: [build-jvm-macOS-M1-native-lib]
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-20.04
```

Comment:
Should the `publish-android`, `publish-python` and `test-python` or any other places with `ubuntu-22.04` also be changed to `ubuntu-20.04`?